WebRoot/js/zimbraMail/prefs/view/ZmZimletsPage.js

```diff
@@ -555,11 +555,17 @@ ZmPrefZimletListView.prototype._getCellId = function(item, field, params) {
 	return DwtId.getListViewItemId(DwtId.WIDGET_ITEM_CELL, "zimlets", item.name, field);
 };
 
+ZmPrefZimletListView.prototype._getCellClass = function (item, field) {
+	if (field === ZmPrefZimletListView.COL_ACTIVE) {
+		return "ZmAlignCenter";
```

Comment:
instead of naming class based on style attribute can we name it based on column name?